### ISSUE TITLE
Integrate slack-accessibility-bot into slack

### ISSUE THREAD

**Issue by christinach** [pulibrary/slack-accessibility-bot#1]
### User story
As a user who is interested in using slack bots for accessibility, I want to be able to install the bot in slack.

### Acceptance criteria
- [ ] I can install the bot in Slack
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...
